custom_components/kia_uvo/Vehicle.py

```diff
@@ -64,7 +65,16 @@ async def update(self):
                     self.set_geocoded_location, current_vehicle_location
                 )
 
-            async_dispatcher_send(self.hass, self.topic_update)
+            if (
+                self.get_child_value("vehicleStatus.engine") == False
+                and previous_vehicle_status.engine == False
```

Comment:
Does this line really work? Have you tried this?

Reply:
@fuatakgun nope that syntax is my old ruby showing, I've updated it to correct python syntax and verified it causes no regressions anymore locally, it will take a day or two in order to verify fully